.teamcity/settings.kts

```diff
@@ -0,0 +1,76 @@
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.pullRequests
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.xmlReport
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.XmlReport
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.nuGetPublish
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+import jetbrains.buildServer.configs.kotlin.v2019_2.vcs.GitVcsRoot
+
+version = "2021.2"
+
+project {
+    buildType(Shimgen)
+}
+
+object Shimgen : BuildType({
+    id = AbsoluteId("Shimgen")
+    name = "Build"
+
+    artifactRules = """
+    """.trimIndent()
+
+    params {
+        password("env.EAZFUSCATOR_NET_LICENSE", "credentialsJSON:cdac4820-7cd7-54e4-b539-8ecaa3490a3c", display = ParameterDisplay.HIDDEN, readOnly = true)
+        param("env.vcsroot.branch", "%vcsroot.branch%")
+        param("env.Git_Branch", "%teamcity.build.vcs.branch.Shimgen_ShimgenVcsRoot%")
+        param("teamcity.git.fetchAllHeads", "true")
+    }
+
+    vcs {
+        root(DslContext.settingsRoot)
+
+        branchFilter = """
+            +:*
+        """.trimIndent()
+    }
+
+    steps {
+        powerShell {
+            name = "Prerequisites"
+            scriptMode = script {
+                content = """
+                    choco install windows-sdk-7.1 netfx-4.8-devpack visualstudio2022-workload-manageddesktopbuildtools --confirm --no-progress
+                    exit ${'$'}LastExitCode
+                """.trimIndent()
+            }
+        }
+
+        step {
+            name = "Include Signing Keys"
+            type = "PrepareSigningEnvironment"
+        }
+
+        script {
+            name = "Call Cake"
+            scriptContent = "call build.official.bat --verbosity=diagnostic --target=CI --testExecutionType=all --shouldRunOpenCover=false"
+        }
+    }
+
+    triggers {
+        vcs {
+            branchFilter = ""
+        }
+    }
+
+    features {
+        pullRequests {
+            provider = gitlab {
+                authType = token {
+                    token = "%system.GitLabPAT%"
+                }
+            }
+        }
+    }
```

Comment:
I presume this was copied from the Shimgen config, and since this is GitHub likely doesn't need the GitLab configuration...

Reply:
I was very confused to see your comment here, as I _know_ I changed this! Turns out that there were some unsaved changes in my editor, that hadn't made it into the commit.  I will get this addressed.